### PR TITLE
Reduce complexity of ThreeDSurfacePlot example

### PR DESCRIPTION
**Pull request by kolibril13**
Currently, we get from time to time the message:
```
Command killed due to excessive memory consumption
Command time: 775s Return: 137
```
from read the docs build.
I think we should speed up the building process a bit, and my idea is to render only the last frame from the 3D examples.
EDIT:
After this pr: Build took 529 seconds

### CODE REVIEW

**Comment by behackl** [comment]
While the 3d examples certainly do take a comparatively long time to render, simply removing the animation from them doesn't feel like a step in the right direction to me. 

Manim can be used for 3D animations, so we should also properly showcase that. What I do agree with, however, is that I think it would make sense to replace the current 3D examples (in particular ThreeDCameraRotation, and ThreeDCameraIllusionRotation) with ones that illustrate Manim's 3D capabilities a bit better; I've always found two so similar examples in the gallery a bit odd.

**Comment by kolibril13** [comment]
In general, I agree, there should be some 3d examples.
However, I find it a bit unnecessary that they are rendered for every commit that is done, and it readthedocs is currently the longest build in our pipeline.
Is there maybe an option in the manim_directive to render these videos for the release version o f the docs and only save the last frame for commits in pull requests?
That would be a very elegant solution.

**Comment by behackl** [comment]
> In general, I agree, there should be some 3d examples.
> However, I find it a bit unnecessary that they are rendered for every commit that is done, and it readthedocs is currently the longest build in our pipeline.

Given our not too great test coverage, the documentation build also tests whether the changes from a PR still allow currently existing scenes render successfully; I can remember *multiple* occasions where problems were detected only due to build failures in the documentation.

> Is there maybe an option in the manim_directive to render these videos for the release version o f the docs and only save the last frame for commits in pull requests?
> That would be a very elegant solution.

We could look into that and I'd be somewhat confident that we could get it to work -- but at the same time this would mean that we couldn't properly preview changes of PRs adding new examples to the documentation any longer. I'd be interested what other devs think about this.

**Comment by jsonvillanueva** [comment]
I think the issue with RTD is that we can only have 2 concurrent builds active. When we can get our manim_directive.py to render scenes with `renderer=opengl`, I think this will drastically cut down the time for RTD and make time less of an issue. I don't think we should outright remove 3D examples though unless they're too similar/not needed. I agree that some could be images, but we can *and should* have a few videos for 3D animations.

Sometimes the docs are built unnecessarily though. In the past, we've run into concurrency issues because multiple PRs are waiting in line for RTD and don't actually need to render the docs. In these cases, it would be nice to manually kick the PR from using RTD so others can use it, but I think for now we'll just have to wait the 588~657 seconds that RTD usually takes.

File: docs/source/examples.rst [hunk 1]
```diff
@@ -489,7 +489,9 @@ Special Camera Settings
             self.set_camera_orientation(phi=75 * DEGREES, theta=30 * DEGREES)
             self.add(axes, sphere)
 
+
 .. manim:: ThreeDCameraRotation
+    :save_last_frame:
```
Comment (kolibril13) [inline docs/source/examples.rst:494]:
```suggestion
```

**Comment by kolibril13** [comment]
ok, I've now updated only the ThreeDSurfacePlot example:
image

**Review by jsonvillanueva — APPROVED** [review]
I'm okay with making ThreeDSurfacePlot an image and keeping the less complex ThreeDCameraRotation a video. LGTM!

**Comment by jsonvillanueva** [comment]
@jsonvillanueva please remember to label/make the PR name more descriptive before merging... oh wait that's me.

**Pull request merged**